openmetadata-ui/src/main/resources/ui/src/pages/ContainerPage/ContainerPage.tsx

```diff
@@ -90,7 +90,9 @@ const ContainerPage = () => {
   const [hasError, setHasError] = useState<boolean>(false);
   const [isEditDescription, setIsEditDescription] = useState<boolean>(false);
 
-  const [containerData, setContainerData] = useState<Container>();
+  const [containerData, setContainerData] = useState<Container>(
+    {} as Container
+  );
```

Comment:
why are we typecasting this? it can lead to unexpected results!

Reply:
@ShaileshParmar11  reverted these changes!